reconcile/utils/openshift_resource.py

```diff
@@ -59,7 +59,7 @@ def obj_intersect_equal(self, obj1, obj2):
                     pass
                 elif self.ignorable_key_value_pair(obj1_k, obj1_v):
                     pass
-                elif obj1_k in ['data', 'labels']:
+                elif obj1_k in ['data', 'labels', 'annotations']:
```

Comment:
You know how much I like sets, this looks to me like (warning: untested!):
```python
diff = set(keys(obj2_v)) - (set(keys(obj1_v) - set(IGNORABLE_DATA_FIELDS))
```

... but I guess that requires more changes than this one-liner.